packages/smtml/smtml.0.2.5/opam

```diff
@@ -0,0 +1,57 @@
+opam-version: "2.0"
+synopsis: "A Front-end library for SMT solvers in OCaml"
+description: "A Multi Back-end Front-end for SMT Solvers in OCaml."
+maintainer: "Filipe Marques <filipe.s.marques@tecnico.ulisboa.pt>"
+authors: "Filipe Marques <filipe.s.marques@tecnico.ulisboa.pt>"
+license: "GPL-3.0-only"
+homepage: "https://github.com/formalsec/smtml"
+doc: "https://formalsec.github.io/smtml/smtml/index.html"
+bug-reports: "https://github.com/formalsec/smtml/issues"
+depends: [
+  "dune" {>= "3.10"}
+  "ocaml" {>= "4.14.0"}
+  "prelude" {>= "0.3"}
+  "ocaml_intrinsics"
+  "fmt" {>= "0.8.7"}
+  "cmdliner" {>= "1.2.0"}
+  "zarith" {>= "1.5"}
+  "hc" {>= "0.3"}
+  "menhir" {build & >= "20220210"}
+  "rusage"
+  "yojson" {>= "1.6.0"}
+  "odoc" {with-doc}
+  "bisect_ppx" {with-test & >= "2.5.0"}
+]
+depopts: [
+  "z3" {>= "4.12.2" & < "4.14"}
+  "colibri2"
+  "bitwuzla-cxx" {>= "0.4.0"}
+  "cvc5"
+]
```

Comment:
```suggestion
depopts: [
  "z3"
  "colibri2"
  "bitwuzla-cxx"
  "cvc5"
]
```

This information should only be in the `conflicts` section. (As per advice on https://github.com/ocaml/opam-repository/wiki/PR-checks )